### ISSUE TITLE
OpenSlides fails in Internet Explorer 11

### ISSUE THREAD

**Issue by emanuelschuetze** [OpenSlides/OpenSlides#2090]
Try to open OpenSlides 2.0 in IE 11. Fails with JS error:
`Die Eigenschaft "src" eines undefinierten oder Nullverweises kann nicht abgerufen werden.
Datei: openslides-libs.js, Zeile: 36, Spalte: 16243`

Maybe the lib of pdf.js or angular-pdf can't support IE11.
MS Edge browser (on Win10) works successfully.

**Comment by FinnStutzenstein** [comment]
Some research is done:

The [angular wrapper](https://github.com/sayanee/angularjs-pdf) for pdf.js generates the error in [this](https://github.com/sayanee/angularjs-pdf/blob/master/example/js/lib/pdf.js#L9172) line. Referencing the [MDN](https://developer.mozilla.org/en/docs/Web/API/Document/currentScript) IE does not support `document.currentScript`. There are polyfill solutions (e.g. [Polyfill from JamesMGreene](https://github.com/JamesMGreene/document.currentScript)) but will not work for IE11 (see description from JamesMGreene).

Another [wrapper](https://github.com/legalthings/angular-pdfjs-viewer) I discoverd work with chrome, firefox and IE11, but this is not that, what we want. Quote: "Embed PDF.js viewer into your angular application, maintaining that look and feel of pdf's we all love. The directive embeds the full viewer[...]". The controls and pdf canvas could not be splitted up, so the controls would be on the projector.

The other solution would be to only use pdf.js and create our own wrapper/implementation of pdf.js. This would take a lot of time and development resources, so I took a quick look in the angular-pdf source code.

Replacing the inner function where the `pdfJsSrc` is evaluated to return an empty string works for me in chrome, FF, and IE11:
```
if (!PDFJS.workerSrc && typeof document !== 'undefined') {                                                                            
   // workerSrc is not set -- using last script url to define default location                                                         
   PDFJS.workerSrc = "";                                                                                               
 }
```
So creating a fork and just changing this part could be a fix for that. But this should be verified by more persons to ensure the fix.

(Remark: if this fix works more code could be removed..)

**Comment by emanuelschuetze** [comment]
angular-pdf has released v1.3.1 6 days ago! This release fixed this bug. OpenSlides runs successfully in IE11. Nice.
